Makefile.in

```diff
@@ -63,6 +66,13 @@ distcheck:
 	@$(ECHO) @PACKAGE_VERSION@
 
 dist: config.h.in configure
-	tar -s +^+msktutil-@PACKAGE_VERSION@/+ -jcvf msktutil-@PACKAGE_VERSION@.tar.bz2 \
+	$(TAR) --transform='s+^+msktutil-@PACKAGE_VERSION@/+' -jcvf msktutil-@PACKAGE_VERSION@.tar.bz2 \
                 $(SOURCES) $(HEADERS) configure README LICENSE ChangeLog INSTALL \
-                Makefile.in configure.ac config.h.in msktutil.M autogen.sh
+                Makefile.in configure.ac config.h.in msktutil.M autogen.sh windows-salt.txt msktutil.spec.in $(SPEC)
```

Comment:
windows-salt.txt is mostly interesting to developers. Not sure if we really want to put it into the distribution tarball.